flower/events.py

```diff
@@ -24,30 +24,39 @@
 
 
 class PrometheusMetrics(object):
-    events = PrometheusCounter('flower_events_total', "Number of events", ['worker', 'type', 'task'])
-
-    runtime = Histogram(
-        'flower_task_runtime_seconds',
-        "Task runtime",
-        ['worker', 'task'],
-        buckets=options.task_runtime_metric_buckets
-    )
-    prefetch_time = Gauge(
-        'flower_task_prefetch_time_seconds',
-        "The time the task spent waiting at the celery worker to be executed.",
-        ['worker', 'task']
-    )
-    number_of_prefetched_tasks = Gauge(
-        'flower_worker_prefetched_tasks',
-        'Number of tasks of given type prefetched at a worker',
-        ['worker', 'task']
-    )
-    worker_online = Gauge('flower_worker_online', "Worker online status", ['worker'])
-    worker_number_of_currently_executing_tasks = Gauge(
-        'flower_worker_number_of_currently_executing_tasks',
-        "Number of tasks currently executing at a worker",
-        ['worker']
-    )
+    __instance = None
+
+    @staticmethod
```

Comment:
I think this should be a class method. I don't expect us to inherit from this class, but then it in principle would work for any class based on this one.

```python
    @classmethod
    def get_instance(cls):
        if cls.__instance is None:
            cls.__instance = cls()
        return cls.__instance
```

an alternative I guess is to have a global variable which gets only set once when we need it in EventsState and then use a factory method. @mher any preferrences?

Reply:
@Tomasz-Kluczkowski I think second idea is enough. I have implemented the same. please have a look